gemfiles/rails_5.2.gemfile

```diff
@@ -6,4 +6,6 @@ gem "activerecord", "5.2.8.1"
 gem "activesupport", "5.2.8.1"
 gem "rails", "5.2.8.1"
 
+gem "sqlite3", "1.5.4" if RUBY_VERSION < "2.7"
```

Comment:
The `gemfiles/*.gemfile` files are generated by [appraisals](https://github.com/thoughtbot/appraisal). Can you update the [Appraisals](https://github.com/salsify/goldiloader/blob/4-x-stable/Appraisals) file so these gemfiles can be regenerated via `appraisal generate`?

Reply:
I tried to do that with the [install_if](https://github.com/thoughtbot/appraisal/pull/176) option, but that failed for older ruby versions.
Can you give me a hint on where `appraisal generate` is actually being evaluated? It seems this is not during the CI build but locally before committing, so the conditional checking for the `RUBY_VERSION` cannot got into the Appraisals, right? Or am I missing sth?

Reply:
Correct we run `appraisal generate` locally and and checkin the generated gemfiles whenever we modify `Appraisals`. Leveraging the `install_if` option in `Appraisals` looks like it should work.